pages/common/autossh.md

```diff
@@ -0,0 +1,21 @@
+# autossh
+
+> Runs, monitors and restarts SSH connections.
+> Auto-reconnects to keep port forwarding tunnels up. Accepts all ssh flags.
+
+- Open an SSH connection with port forward monitoring and forward a local port to a remote one.
+
+`autossh -M {{monitor_port}} {{ssh_command}}`
+`autossh -M {{monitor_port}} -L {{local_port}}:localhost:{{remote_port}} {{user}}@{{host}}`
+
+- Fork before executing ssh (runs in the background), no remote shell.
+
+`autossh -f -M {{monitor_port}} -N {{ssh_command}}`
+
+- Log at maximum verbosity to a file from autossh running in the background.
+
+`AUTOSSH_LOGLEVEL=7 AUTOSSH_LOGFILE={{log_file}} -f -M {{monitor_port}} -N {{ssh_command}}`
+
+- No monitor port, vebose ssh output, no remote shell, require a successful local port forward, send keep-alives every 10 secs and disconnecting if 3 in a row fail.
```

Comment:
I can, but my intention was not to introduce log-level and the logfile individually, but to suggest a command sufficient for troubleshooting.  The default behavior logs only to syslog but in the unix tradition is basically silent unless there's actually an error so if you're troubleshooting you'll likely need both.  If I were to introduce a sixth example I would sooner include yet another SSH flag for writing SSH to a logfile instead of stderr but was trying really hard to highlight autossh options, not ssh ones.